main.tf

```diff
@@ -170,14 +194,6 @@ resource "aws_eks_node_group" "default" {
     version = local.launch_template.latest_version
   }
 
-  dynamic "remote_access" {
```

Comment:
this block is not supported when using launch templates. currently breaks the entire module if `ec2_ssh_key` or `source_security_group_ids` is specified